CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Change Log for corePKCS11 Library
 
+## v3.0.0 (February 2021)
+* Removed default `PKCS11_PAL_DestroyObject` implementation from `core_pkcs11_mbedtls.c`
```

Comment:
We are providing links to PRs in other repository CHANGELOGs (example of [Jobs CHANGELOG](https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/blob/main/CHANGELOG.md#v110-march-2021)). It can be a good practice for this repository to adopt as well. 🙂